codeJE/dpl-config.json

```diff
@@ -0,0 +1,562 @@
+{
+    "internal-dpl-clock": "",
+    "internal-dpl-aod-reader": {
+        "time-limit": "0",
+        "orbit-offset-enumeration": "0",
+        "orbit-multiplier-enumeration": "0",
+        "start-value-enumeration": "0",
+        "end-value-enumeration": "-1",
+        "step-value-enumeration": "1",
+        "aod-file": "@list_o2.txt"
+    },
+    "internal-dpl-aod-spawner": "",
+    "timestamp-task": {
+        "verbose": "false",
+        "rct-path": "RCT/Info/RunInformation",
+        "orbit-reset-path": "CTP/Calib/OrbitReset",
+        "ccdb-url": "http://alice-ccdb.cern.ch",
+        "isRun2MC": "false"
+    },
+    "zdc-converter": "",
+    "internal-dpl-aod-index-builder": "",
+    "track-extension": {
+        "compatibilityIU": "false",
+        "processRun2": "true",
+        "processRun3": "false"
+    },
+    "bc-selection-task": {
+        "triggerBcShift": "999",
+        "processRun2": "true",
+        "processRun3": "false"
+    },
+    "track-selection": {
+        "isRun3": "false",
+        "produceTable": "-1",
+        "produceFBextendedTable": "-1",
+        "compatibilityIU": "false",
+        "itsMatching": "0",
+        "dcaSetup": "0",
+        "ptMin": "0.100000001",
+        "ptMax": "1e+10",
+        "etaMin": "-0.800000012",
+        "etaMax": "0.800000012"
```

Comment:
Fix these trailing digits. They should not appear in the provided configuration.